src/walletSdk/Types/sep12.ts

```diff
@@ -0,0 +1,46 @@
+export type CustomerInfoMap = {
+  [key: string]: string;
+};
+
+export enum Sep12Status {
+  ACCEPTED = "ACCEPTED",
+  PROCESSING = "PROCESSING",
+  NEEDS_INFO = "NEEDS_INFO",
+  REJECTED = "REJECTED",
+  VERIFICATION_REQUIRED = "VERIFICATION_REQUIRED",
+}
+
+export enum Sep12Type {
+  string = "string",
+  binary = "binary",
+  number = "number",
+  date = "date",
+}
+
+export type Field = {
+  type: Sep12Type;
```

Comment:
@Ifropc kotlin has a couple of these fields optional (eg. https://github.com/stellar/kotlin-wallet-sdk/blob/3b81495e021939153d7f713a666b081ebbe6a61c/wallet-sdk/src/main/kotlin/org/stellar/walletsdk/customer/Data.kt#L21-L22)

according to the [SEP](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0012.md#fields) they should be required, lmk if there was a reason to make them optional though and I can change these

Reply:
I don't remember why they were left optional. It was either by mistake, or because testanchor didn't provide them for some reason

Reply:
If it's always provided by the testanchor I don't see why we shouldn't make them mandatory